install.md

```diff
@@ -318,11 +318,18 @@ To add build tags to the make option the `BUILDTAGS` variable must be set, for e
 make BUILDTAGS='seccomp apparmor'
 ```
 
-| Build Tag | Feature                            | Dependency  |
-|-----------|------------------------------------|-------------|
-| seccomp   | syscall filtering                  | libseccomp  |
-| selinux   | selinux process and mount labeling | libselinux  |
-| apparmor  | apparmor profile support           | libapparmor |
+| Build Tag                        | Feature                            | Dependency           |
+|----------------------------------|------------------------------------|----------------------|
+| apparmor                         | apparmor support                   | libapparmor          |
+| exclude_graphdriver_btrfs        | exclude btrfs                      | libbtrfs             |
+| exclude_graphdriver_devicemapper | exclude device-mapper              | libdm                |
```

Comment:
This one is basically mandatory, so we should mark it as such

Reply:
Why is it mandatory?

Reply:
The driver's locking can't handle podman's multiple processes - you can do very bad things because of this.

Reply:
So, while you can produce a working build without the tag, using the graphdriver backend will never be supported, voids your warranty, etc.

Reply:
:+1: I'll add a note for that. Thanks for checking!

Reply:
NEVER is a not how I would state it.  @nalind has some ideas of how we could fix this so that multiple processes could use devicemapper at the same time, but has not had time or priority to implement it.